src/storageClients/BlobStorageManager.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+const Promise = require('bluebird');
+const azure = Promise.promisifyAll(require('azure-storage'));
```

Comment:
this seems like an expensive operation as the azure-storage library is a large one. I'd prefer to write the promise code out.